tests/test_z_download.py

```diff
@@ -0,0 +1,10 @@
+import io
+
+def test_Download(test_client):
+    with open("result.abc", "rb") as abc:
+        abcStringIO = io.BytesIO(abc.read())
+
+    response = test_client.get('/return-files',
```

Comment:
I don't see a download request here, kind of puzzled by what this is testing? Is this testing the behavior that you upload and then download from the resulted page?